### DISCUSSION TITLE
Supabase Remote Config like Firebase

### DISCUSSION THREAD

**Discussion by ThomasDevApps** [supabase/supabase#23323] [Feature Requests] [answered]
Hi, I wanted to ask you if Supabase will have a "Remote Config" feature like Firebase ?

In a Flutter project, this could be very useful for storing information like :

- `min_build_ios `
- `min_build_android`

But it could be even more interesting for trunk-based development projects, by facilitating feature flags, for example :

- `enable_feature_1` : false
- `enable_feature_2` : true

Of course, these values can be changed at any time.

The value of the key present in the remote config could be, for example : 
- a boolean 
- a string
- an int
- a JSON

**Comment by harpreetslabs** [comment]
@ThomasDevApps  did you find an answer?

**Reply by ThomasDevApps** [reply]
@harpreetslabs no, not yet...

**Reply by dshukertjr** [reply, marked as answer]
We do not have anything like this planned at the moment, but you can certainly create your own table within a Supabase database to store these key-value pairs and cache it on device!

**Reply by OlivierCo** [reply]
@dshukertjr you are absolutely right. It's not rocket science.

**Reply by rakesh-ayyaswami** [reply]
There's more to feature flags than just a key-value table. I wish Supabase could support the [Open Feature](https://openfeature.dev/) standard.
